File: 231128/정수 명령 처리 5/process-numeric-commands-5.py
# 변수 선언 및 입력
n = int(input())

# 동적 배열 선언
v = list()

for _ in range(n):
    command = input()

    if command.startswith("push_back"):
        _, num = tuple(command.split())
        v.append(int(num))
    elif command.startswith("pop_back"):
        v.pop()
    elif command.startswith("size"):
        print(len(v))
    else:
        _, index = tuple(command.split())
        print(v[int(index) - 1])

File: 231128/정수 명령 처리 5/process-numeric-commands-5.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

vector <int> v;
int n, a;
string cmd;

int main() {
    cin >> n;
    while(n--){
        cin >> cmd;
        if (cmd.compare("push_back") == 0){
            cin >> a;
            v.push_back(a);
        }
        else if (cmd.compare("pop_back") == 0){
            v.pop_back();
        }
        else if (cmd.compare("size") == 0){
            cout << v.size() << '\n';
        }
        else if (cmd.compare("get") == 0){
            cin >> a;
            cout << v[a - 1] << '\n';
        }
    }
    return 0;
}